src/test/java/uk/gov/companieshouse/model/validator/RelationshipValidatorTest.java

```diff
@@ -6,14 +6,14 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.InjectMocks;
-import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.mockito.junit.jupiter.MockitoExtension;
 import uk.gov.companieshouse.TestData;
 import uk.gov.companieshouse.TestUtil;
 import uk.gov.companieshouse.model.Appeal;
 import uk.gov.companieshouse.model.CreatedBy;
 import uk.gov.companieshouse.model.Reason;
 
-@ExtendWith(SpringExtension.class)
+@ExtendWith(MockitoExtension.class)
 class RelationshipValidatorTest {
```

Comment:
We use the **@DisplayName** annotation on test methods to give a more verbose way of describing a test. Can this annotation be added to test methods in the class please.

Reply:
Done in new commit